tests/create-release.test.js

```diff
@@ -57,7 +58,7 @@ describe('Create Release', () => {
       .mockReturnValueOnce('myRelease')
       .mockReturnValueOnce('myBody')
       .mockReturnValueOnce('false')
-      .mockReturnValueOnce('false');
+      .mockReturnValueOnce('main');
```

Comment:
I am not sure what this change did to the `jest` framework. You removed one mocked property and added a new one for the `target_commitish` what happened to the previous value?

Reply:
In fact the last `mockReturn` was never used by the test in this case. They removed an input but did not adapted the tests.